hack/common.sh

```diff
@@ -658,6 +658,25 @@ EOF
 }
 readonly -f os::build::save_version_vars
 
+# os::build::get_product_vars exports variables that we expect to change
+# depending on the distribution of Origin
+function os::build::get_product_vars() {
+  if [[ "${OS_ROOT}" == *origin* ]]; then
```

Comment:
This was the question I posed you, though -- do we expect someone to provide it at build-time (and carry the diff in OSE `hack/common` script) or do we try to auto-detect here and carry no diff?